README.md

```diff
@@ -1,76 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# practice-tdd
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+> This app was built with the purpose to practice Test Driven Development.
 
 ## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
+- Ruby
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
 ### Setup
 
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+To get a local copy up and running follow these simple steps.
 
+To setup the project in your local, in the repo page:
+click on code (dropdown list) > Download as ZIP.
+or open terminal of path you want to install project and run this command <br>
+`git clone git@github.com:nicupop729/practice-tdd.git`
```

Comment:
- [ ] Kindly add instructions on how to run the project and the tests locally. Also, because you are using simple_cov, it would be better to mention that it should be installed.